app/views/uploads/show.html.erb

```diff
@@ -66,3 +66,54 @@
     <%= @upload.comment %>
   </div>
 </div>
+
+<div class="container" style="float: left;">
```

Comment:
Should this be broken out into a partial since it is displayed in both new and show?